deps/Makefile

```diff
@@ -159,7 +159,7 @@ lz4: lz4/lz4/lib/liblz4.a
 clickhouse-cpp/clickhouse-cpp/clickhouse/libclickhouse-cpp-lib-static.a:
 	cd clickhouse-cpp && rm -rf clickhouse-cpp-*/ || true
 	cd clickhouse-cpp && tar -zxf v2.3.0.tar.gz
-	cd clickhouse-cpp && ln -fs clickhouse-cpp-*/ clickhouse-cpp
+	cd clickhouse-cpp && ln -fsT clickhouse-cpp-*/ clickhouse-cpp
```

Comment:
As before,

Why do we create symlinks during `make` ? symlinks should be well defined